src/main/java/com/epam/javacourses2016/task16/IFile.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.javacourses2016.task16;
+
+import com.epam.javacourses2016.Point2D;
+import com.epam.javacourses2016.task15.Line;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * Created by Рамиль on 30.11.2016.
+ */
+public class IFile implements SolverTask16.IFileWithPoints {
```

Comment:
_I_ prefix means _Interface_, actually it is not common for Java more C++ style. But anyway it should not be in naming of classes.